terraform/modules/shuttle/systemd/system/shuttle-pg.service.tftpl

```diff
@@ -11,7 +11,7 @@ Restart=always
 ExecStartPre=/usr/bin/docker pull postgres:11
 ExecStart=/usr/bin/docker run --rm \
 			      --network shuttle-net \
-			      --name pg \
+			      --name postgres \
```

Comment:
I'm not sure why this is necessary for `v0.6.0rc1`? Or was this done to stay consistent across?

These tf files are now obsolete, the new ones being in `terraform-aws-shuttle`.

Reply:
Ah! I'll leave this out then.